matrix/common/constants.py

```diff
@@ -15,3 +15,117 @@ class MatrixRequestStatus(Enum):
     COMPLETE = "Complete"
     IN_PROGRESS = "In Progress"
     FAILED = "Failed"
+
+
+CREATE_QUERY_TEMPLATE = {
```

Comment:
Do we need to manage a migration framework? Or will we just rebuild the entire database anytime we need to make changes to this schema?

Reply:
Maybe we can revisit this question after getting performance/load times with `dcplib/ETL` parallelizing the transformers as well. Currently, a download of integration (~10K bundles) completes within an hour, but the single-threaded execution of transformers takes ~7 hours. I'd imagine we'd be below 1 or 2 hours with the full parallelization

Reply:
After a quick search, it looks like [migrating tables to a new schema](https://stackoverflow.com/questions/22548928/how-to-change-table-schema-after-created-in-redshift) may be simple.

Reply:
Cool I'm good with not worrying about this at the moment